git_hooks/README.md

```diff
@@ -0,0 +1,14 @@
+README for git hooks script
+===========================
+git has a way to run scripts, which are invoked by specific git commands.
+The git hooks are located in <root>.git/hooks, and as such are not under version control
+for more information, see https://git-scm.com/docs/githooks
```

Comment:
Style: perhaps use the markdown syntax instead of just writing the plain URL? For example:
```
for more information, see the [git documentation](https://git-scm.com/docs/githooks).
```
will look like this:

for more information, see the [git documentation](https://git-scm.com/docs/githooks).